tools/console/console.js

```diff
@@ -309,7 +309,7 @@ var ProgressDisplayFull = function (console) {
   self._progressBarRenderer = new ProgressBarRenderer(PROGRESS_BAR_FORMAT, options);
   self._progressBarRenderer.start = new Date();
 
-  self._headless = false;
+  self._headless = !!(process.env.METEOR_HEADLESS && process.env.METEOR_HEADLESS != '0');
```

Comment:
I appreciate the consistency, but I was just about to link to the same `METEOR_ALLOW_SUPERUSER` example that @abernix mentioned above, because I think it's important to support e.g. `METEOR_HEADLESS=false`.